libsrc/telestrat/gotoxy.s

```diff
@@ -5,10 +5,9 @@
 ; void gotoxy (unsigned char x, unsigned char y);
 ;
 
-        .export         gotoxy, _gotoxy
+        .export         gotoxy, _gotoxy, _update_adscr
```

Comment:
Do you want `update_adscr()` to be used by C code?  Does the name need that leading underscore?

Reply:
No, update_adscr is not available from C.  I corrected the mistake. Fixed. Thanks.